api/services/elastigroup/aws/emr/schemas/compute/group-commons/capacity.yaml

```diff
@@ -15,3 +15,7 @@ properties:
     type: integer
     description: Maximum number of instances in group
     example: 10
+  unit:
```

Comment:
@stevenfeltner  for each MrScaler, under ‘compute’,  there are ‘instanceGroups’ & ‘instanceWeights’.
are you sure we need to add ‘unit’ under the ‘instanceGroups’->core/task->capacity ?

Reply:
@sagishoshan Correct it will be under the 'instanceGroups' object